Sasquatch/Sasquatch.xcodeproj/project.pbxproj

```diff
@@ -2237,104 +2170,142 @@
 /* End PBXResourcesBuildPhase section */
 
 /* Begin PBXShellScriptBuildPhase section */
-		8099A54F21121501008E1466 /* Verify no build settings */ = {
+		8099A54F21121501008E1466 /* Verify No Build Settings */ = {
 			isa = PBXShellScriptBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
 			);
 			inputPaths = (
 			);
-			name = "Verify no build settings";
+			name = "Verify No Build Settings";
 			outputPaths = (
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
 			shellScript = "\"${SRCROOT}/../Scripts/VerifyNoBuildSettings.swift\" ${PROJECT_NAME}.xcodeproj/project.pbxproj\n";
 			showEnvVarsInLog = 0;
 		};
-		8099A5D62112198F008E1466 /* Verify no build settings */ = {
+		8099A5D62112198F008E1466 /* Verify No Build Settings */ = {
 			isa = PBXShellScriptBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
 			);
 			inputPaths = (
 			);
-			name = "Verify no build settings";
+			name = "Verify No Build Settings";
 			outputPaths = (
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
 			shellScript = "\"${SRCROOT}/../Scripts/VerifyNoBuildSettings.swift\" ${PROJECT_NAME}.xcodeproj/project.pbxproj\n";
 			showEnvVarsInLog = 0;
 		};
-		F84923BB21119AE100E14DBE /* Verify no buildSettings */ = {
+		F83D7A802317C4FF00E7E8F4 /* Remove the wrong framework */ = {
 			isa = PBXShellScriptBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
 			);
+			inputFileListPaths = (
+			);
 			inputPaths = (
 			);
-			name = "Verify no buildSettings";
+			name = "Remove the wrong framework";
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "# We test the app on the fat fake framework produced by our script. \n# In order for linker to not pick up the simple static framework\n# (which it does by default), we need to remove it.\nrm -rf \"${BUILT_PRODUCTS_DIR}\"/AppCenter*.framework\n";
```

Comment:
Why does it bother? Isn't it also in framework search path?

Reply:
It matters. We spent more than a day trying to fix Sasquatch and that's the only way for now. Looking at the logs, during linking proecss, it first looks into this ^ folder and only then in custom one (`AppCenter-SDK-Apple/...`).